docs/source/gs-processing/developer/input-configuration.rst

```diff
@@ -447,6 +447,15 @@ arguments.
         will be considered as an array. For Parquet files, if the input type is ArrayType(StringType()), then the
         separator is ignored; if it is StringType(), it will apply same logic as in CSV.
 
+-  ``huggingface``
+
+   -  Transforms a text feature column to tokens or embeddings with different Hugging Face models, enabling nuanced understanding and processing of natural language data.
+   -  ``kwargs``:
+
+      - ``normalizer`` (String, required): It should be "tokenize_hf".
```

Comment:
Where does the normalizer name come from? This is a text to either embedding or token transformation right? Why do we only accept one value here?

Reply:
It is used to differ the embedding or token transformation, but currently we do not implement embedding transformation for this PR, so we only accept token transformation for now.

Reply:
I would prefer this to be more descriptive then, let's use `action` instead of `normalizer`. 

So for now the only supported `action` will be `tokenize_hf`, and in the future we'll add `embedding_hf`.

Reply:
```suggestion
      - ``action`` (String, required): The action to perform on the text data. Currently we only support text tokenization through HuggingFace models, so the only accepted value here is "tokenize_hf".
```